approval_frame/forms.py

```diff
@@ -17,3 +18,22 @@ class RegistrationFormCustom(RegistrationFormUniqueEmail):
         error_messages={'invalid': _("This value may contain only letters, "
             + "numbers and ./+/-/_ characters.")}
     )
+
+class PasswordResetFormCustom(PasswordResetForm):
```

Comment:
I agree that providing an error message to the user would provide a good user experience. Usually Django follows best practices and I was wondering why they haven't included this feature. So, I dug around a bit and found [this answer](http://stackoverflow.com/questions/22582066/why-does-password-reset-works-with-unregistered-email-in-django) on stack overflow. It looks like the absence of this feature is intentional for security reasons.

More info about best practices for a secure password reset [here] (https://www.fishnetsecurity.com/6labs/resource-library/white-paper/best-practices-secure-forgot-password-feature)

TL;DR Password reset application form should not indicate if the username was valid or not and we shouldn't send the username in the password reset email.